docs/static/advanced-pipeline.asciidoc

```diff
@@ -117,15 +118,8 @@ filter {
     }
 }
 --------------------------------------------------------------------------------
-And change output to see a result :
-[source,json]
---------------------------------------------------------------------------------
-output {
-  stdout { codec => rubydebug }
-}
---------------------------------------------------------------------------------
```

Comment:
I think this is correct because it asks the user only to replace the filter section now as opposed to the entire file.